connectivity/factory/factory.go

```diff
@@ -0,0 +1,248 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package factory
+
+import (
+	_ "embed"
+
+	"github.com/blang/semver/v4"
+
+	"github.com/cilium/cilium-cli/connectivity/check"
+	"github.com/cilium/cilium-cli/connectivity/factory/manifests/template"
+	"github.com/cilium/cilium-cli/utils/features"
+)
+
+var (
+	//go:embed manifests/allow-all-egress.yaml
+	allowAllEgressPolicyYAML string
+
+	//go:embed manifests/allow-all-ingress.yaml
+	allowAllIngressPolicyYAML string
+
+	//go:embed manifests/deny-all-ingress.yaml
+	denyAllIngressPolicyYAML string
+
+	//go:embed manifests/deny-cidr.yaml
+	denyCIDRPolicyYAML string
+
+	//go:embed manifests/allow-cluster-entity.yaml
+	allowClusterEntityPolicyYAML string
+
+	//go:embed manifests/client-egress-only-dns.yaml
+	clientEgressOnlyDNSPolicyYAML string
+
+	//go:embed manifests/client-egress-to-fqdns-one-one-one-one.yaml
+	clientEgressToFQDNsCiliumIOPolicyYAML string
+
+	//go:embed manifests/echo-ingress-from-other-client.yaml
+	echoIngressFromOtherClientPolicyYAML string
+
+	//go:embed manifests/client-egress-to-cidr-cp-host-knp.yaml
+	clientEgressToCIDRCPHostPolicyYAML string
+
+	//go:embed manifests/client-egress-to-cidr-external.yaml
+	clientEgressToCIDRExternalPolicyYAML string
+
+	//go:embed manifests/client-egress-to-cidr-external-knp.yaml
+	clientEgressToCIDRExternalPolicyKNPYAML string
+
+	//go:embed manifests/client-egress-to-cidr-k8s.yaml
+	clientEgressToCIDRK8sPolicyYAML string
+
+	//go:embed manifests/client-egress-to-cidr-node-knp.yaml
+	clientEgressToCIDRNodeKNPYAML string
+
+	//go:embed manifests/client-egress-to-cidr-external-deny.yaml
+	clientEgressToCIDRExternalDenyPolicyYAML string
+
+	//go:embed manifests/client-egress-l7-http.yaml
+	clientEgressL7HTTPPolicyYAML string
+
+	//go:embed manifests/client-egress-l7-http-named-port.yaml
+	clientEgressL7HTTPNamedPortPolicyYAML string
+
+	//go:embed manifests/client-egress-l7-tls.yaml
+	clientEgressL7TLSPolicyYAML string
+
+	//go:embed manifests/client-egress-l7-http-matchheader-secret.yaml
+	clientEgressL7HTTPMatchheaderSecretYAML string
+
+	//go:embed manifests/echo-ingress-from-cidr.yaml
+	echoIngressFromCIDRYAML string
+)
+
+var (
+	clientLabel  = map[string]string{"name": "client"}
+	client2Label = map[string]string{"name": "client2"}
+
+	// Test order is important
+	parallelTests = []factory{
```

Comment:
Since this slice is called `parallelTests`, calling out that "order is important", is a bit strange for the reader here.

How about we call this `primaryTests` or similar until the parallelization is implemented?

Reply:
Fixed.